packages/client/src/pages/Staking/components/index.ts

```diff
@@ -1,2 +1,2 @@
 export { default as ValidatorList } from './ValidatorList'
-export { default as Stats } from './Stats'
+export { default as Stats } from './Stats';
```

Comment:
The Client codebase doesn't use semicolons. How did it get there?

Reply:
Probably got there while testing eslint config.